README.md

```diff
@@ -1,5 +1,12 @@
-## Cortex Certifai Examples 
-This directory contains example Jupyter notebooks illustrating how to use Certifai in a notebook environment.
+# Cortex Certifai Examples 
+This directory contains examples and tutorials illustrating how to use Certifai in a notebook environment and CLI.
 
+### Notebooks
+The *notebooks* folder contains Jupiter Lab notebooks which demonstrate how to use various functions of Certifai in a notebook environment. 
+
+### Tutorials
+The *tutorials* folder contains walkthroughs outside of a notebook environment which demonstrate various Certifai functionality.
```

Comment:
Drop "outside of a notebook environment".

Reply:
Done, thanks!